presto-main/src/main/java/com/facebook/presto/sql/planner/plan/PlanNode.java

```diff
@@ -60,7 +60,8 @@
         @JsonSubTypes.Type(value = ExplainAnalyzeNode.class, name = "explainAnalyze"),
         @JsonSubTypes.Type(value = ApplyNode.class, name = "apply"),
         @JsonSubTypes.Type(value = AssignUniqueId.class, name = "assignUniqueId"),
-        @JsonSubTypes.Type(value = LateralJoinNode.class, name = "lateralJoin")})
+        @JsonSubTypes.Type(value = LateralJoinNode.class, name = "lateralJoin"),
+        @JsonSubTypes.Type(value = AnalyzeFinishNode.class, name = "analyzeFinish")})
```

Comment:
add `,\n` before `}` so next time, when adding a new entry, no old line needs to be modified

Reply:
I think the style check will fail on that.

Reply:
I think it wouldn't